sql/ha_partition.h

```diff
@@ -1465,7 +1465,7 @@ class ha_partition final :public handler
 
     virtual int get_foreign_key_list(THD *thd,
     List<FOREIGN_KEY_INFO> *f_key_list)
-    virtual uint referenced_by_foreign_key()
+    virtual bool referenced_by_foreign_key()
```

Comment:
It’s nice that you are updating this comment to be consistent with your change. I’d suggest to make it even more consistent, by removing the `virtual` and adding `override`.